### ISSUE TITLE
label:enhancement Ready to use compiled .js file and/or npm package

### ISSUE THREAD

**Issue by vird** [leanprover/lean.js#5]

**Comment by soonhokong** [comment]
Well.. In the README.md file, I mentioned that the latest version of compiled javascript is available in the following paragraph:

![screen shot 2015-09-06 at 10 55 45 am](https://cloud.githubusercontent.com/assets/403281/9705050/da348c94-5485-11e5-9ebe-a085efd90eec.png)

Please use https://leanprover.github.io/lean.js/lean.js .

**Comment by soonhokong** [comment]
I agree that it would be nice if we have it as a npm package. But for now, I don't have enough time for that. Of course, if anyone is willing to do that, I'm happy to help.
